.gitignore

```diff
@@ -19,3 +19,6 @@ build/
 
 # OSX
 .DS_Store
+dist/public/js/react-d3.min.js
+dist/public/js/react-d3.min.js.map
+test.js
```

Comment:
ugh, I was a little confused. I thought that's from the master. Don't know where that's from. I'll remove it.